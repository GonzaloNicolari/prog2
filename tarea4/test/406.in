# prueba para iterador
reiniciar_iterador
avanzar_iterador
esta_definida_actual
agregar_a_iterador (5,q)
esta_definida_actual
actual_en_iterador
reiniciar_iterador
actual_en_iterador
hay_siguiente_en_iterador
agregar_a_iterador (25,e)
actual_en_iterador
hay_siguiente_en_iterador
reiniciar_iterador
actual_en_iterador
avanzar_iterador
actual_en_iterador
hay_siguiente_en_iterador
reiniciar_iterador
actual_en_iterador
hay_siguiente_en_iterador
agregar_a_iterador (45,qqq)
actual_en_iterador
avanzar_iterador
actual_en_iterador
hay_siguiente_en_iterador
avanzar_iterador
actual_en_iterador
hay_siguiente_en_iterador
avanzar_iterador
agregar_a_iterador (5,fsd)
hay_siguiente_en_iterador
esta_definida_actual
reiniciar_iterador
agregar_a_iterador (12,et)
agregar_a_iterador (75,gh)
agregar_a_iterador (25643,urtu)
agregar_a_iterador (709,cdsf)
agregar_a_iterador (31,rwr)
agregar_a_iterador (5,nte)
agregar_a_iterador (65,cdfew)
agregar_a_iterador (8,yyr)
agregar_a_iterador (65653,brq)
agregar_a_iterador (28,vrfe)
agregar_a_iterador (-8643,qweqw)
agregar_a_iterador (98,nty)
agregar_a_iterador (-42332,brwq)
agregar_a_iterador (98,fwrwe)
bloquear_iterador
agregar_a_iterador (0,++)
actual_en_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
actual_en_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
avanzar_iterador
hay_siguiente_en_iterador
avanzar_iterador
actual_en_iterador
hay_siguiente_en_iterador
actual_en_iterador
avanzar_iterador
agregar_a_iterador (0,--)
hay_siguiente_en_iterador
Fin
