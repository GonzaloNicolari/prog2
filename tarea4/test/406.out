1>#  prueba para iterador.
2>Reiniciado.
3>Avanzando.
4>Actual de it NO está definida.
5>Agregando.
6>Actual de it NO está definida.
7>Actual no definida.
8>Reiniciado.
9>(5,q)
10>No hay siguiente.
11>Agregando.
12>(5,q)
13>Hay siguiente.
14>Reiniciado.
15>(5,q)
16>Avanzando.
17>(25,e)
18>No hay siguiente.
19>Reiniciado.
20>(5,q)
21>Hay siguiente.
22>Agregando.
23>(5,q)
24>Avanzando.
25>(25,e)
26>Hay siguiente.
27>Avanzando.
28>(45,qqq)
29>No hay siguiente.
30>Avanzando.
31>Agregando.
32>Actual no definida.
33>Actual de it NO está definida.
34>Reiniciado.
35>Agregando.
36>Agregando.
37>Agregando.
38>Agregando.
39>Agregando.
40>Agregando.
41>Agregando.
42>Agregando.
43>Agregando.
44>Agregando.
45>Agregando.
46>Agregando.
47>Agregando.
48>Agregando.
49>Iterador bloqueado.
50>Agregando.
51>(5,q)
52>Avanzando.
53>Avanzando.
54>Avanzando.
55>(5,fsd)
56>Avanzando.
57>Avanzando.
58>Avanzando.
59>Avanzando.
60>Avanzando.
61>Avanzando.
62>Avanzando.
63>Avanzando.
64>Avanzando.
65>Avanzando.
66>Avanzando.
67>Avanzando.
68>Avanzando.
69>Hay siguiente.
70>Avanzando.
71>(98,fwrwe)
72>No hay siguiente.
73>(98,fwrwe)
74>Avanzando.
75>Agregando.
76>Actual no definida.
77>Fin.
